reactApp/src/components/authentication/Login.js

```diff
@@ -0,0 +1,9 @@
+function Login() {
+  return (
+    <>
+      <h1>(MODAL) Login with your Naturdoc Account </h1>
```

Comment:
For a login Modal you can use a Dialog MUI component

https://mui.com/material-ui/react-dialog/
https://mui.com/material-ui/react-modal/

Reply:
You can put the navigation bar and the footer into a Higher-Order Component, so you can easily have these fixed elements for all the pages without repeating them everywhere

Reply:
Thank you Soma for the reviews and encouragements! Thank you for the tip, will definitely use higher order components as templates and refactor a bit all this ^_^